--- vorher
+++ nachher
@@ -66,5 +66,5 @@
 <link as="script" crossorigin="" fetchpriority="low" href="https://www.3close.de/wp-content/plugins/preloader-plus//assets/js/preloader-plus.min.js" rel="preload" type="text/javascript"/>
 <link as="style" crossorigin="" fetchpriority="low" href="https://www.3close.de/wp-content/plugins/preloader-plus/assets/css/preloader-plus.min.css" rel="preload" type="text/css"/>
-<!-- This site is optimized with the Yoast SEO plugin v28.4 - https://yoast.com/product/yoast-seo-wordpress/ -->
+<!-- This site is optimized with the Yoast SEO plugin v28.5 - https://yoast.com/product/yoast-seo-wordpress/ -->
 <meta content="A new Shopware plugin and a Windows app that addresses surprisingly frequent frictions are the first two developments I want to share from my workbench today." name="description"/>
 <link href="https://www.3close.de/answers/2026/01/31/a-fresh-start-into-2026-small-utilities-with-a-big-impact/" rel="canonical"/>
