--- vorher
+++ nachher
@@ -66,5 +66,5 @@
 <script id="hitchcock_global-js" src="https://www.ideetrifftfarbe.de/wp-content/themes/hitchcock/assets/js/global.js?ver=2.0.2">
 </script>
-<meta content="WordPress 7.1.1" name="generator"/>
+<meta content="WordPress 7.1.2" name="generator"/>
 <link href="https://www.ideetrifftfarbe.de/vita/" rel="canonical"/>
 <link href="https://www.ideetrifftfarbe.de/?p=379" rel="shortlink"/>
@@ -101,5 +101,5 @@
 }
 </style>
-<link href="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/css/front.css?ver=3.0.6" id="modula-css" media="all" rel="stylesheet"/>
+<link href="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/css/front.css?ver=3.0.8" id="modula-css" media="all" rel="stylesheet"/>
 </head>
 <body class="wp-singular page-template-default page page-id-379 wp-theme-hitchcock post single show-preview-titles modula-best-grid-gallery" data-cmplz="1">
@@ -204,5 +204,5 @@
 </div>
 <div class="modula-items grid-gallery">
-<div class="modula-item">
+<div class="modula-item effect-pufrobo">
 <div class="modula-item-overlay">
 </div>
@@ -219,5 +219,5 @@
 </div>
 </div>
-<div class="modula-item">
+<div class="modula-item effect-pufrobo">
 <div class="modula-item-overlay">
 </div>
@@ -448,9 +448,9 @@
 <script defer="" id="cmplz-cookiebanner-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/complianz-gdpr/cookiebanner/js/complianz.min.js?ver=1788920872">
 </script>
-<script id="modula-isotope-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/isotope.min.js?ver=3.0.6">
-</script>
-<script id="modula-isotope-packery-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/isotope-packery.min.js?ver=3.0.6">
-</script>
-<script id="dompurify-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/purify.min.js?ver=3.0.6">
+<script id="modula-isotope-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/isotope.min.js?ver=3.0.8">
+</script>
+<script id="modula-isotope-packery-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/isotope-packery.min.js?ver=3.0.8">
+</script>
+<script id="dompurify-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/purify.min.js?ver=3.0.8">
 </script>
 <script id="modulaFancybox-js-before">
@@ -480,7 +480,7 @@
 //# sourceURL=modulaFancybox-js-before
 </script>
-<script id="modulaFancybox-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/modula-fancybox.min.js?ver=3.0.6">
-</script>
-<script id="modula-fancybox-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/fancybox.min.js?ver=3.0.6">
+<script id="modulaFancybox-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/modula-fancybox.min.js?ver=3.0.8">
+</script>
+<script id="modula-fancybox-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/fancybox.min.js?ver=3.0.8">
 </script>
 <script id="modula-js-before">
@@ -488,8 +488,8 @@
 //# sourceURL=modula-js-before
 </script>
-<script id="modula-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/jquery-modula.min.js?ver=3.0.6">
+<script id="modula-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/jquery-modula.min.js?ver=3.0.8">
 </script>
 <script id="wp-emoji-settings" type="application/json">
-{"baseUrl":"https://s.w.org/images/core/emoji/17.0.2/72x72/","ext":".png","svgUrl":"https://s.w.org/images/core/emoji/17.0.2/svg/","svgExt":".svg","source":{"concatemoji":"https://www.ideetrifftfarbe.de/wp-includes/js/wp-emoji-release.min.js?ver=7.1.1"}}
+{"baseUrl":"https://s.w.org/images/core/emoji/17.0.2/72x72/","ext":".png","svgUrl":"https://s.w.org/images/core/emoji/17.0.2/svg/","svgExt":".svg","source":{"concatemoji":"https://www.ideetrifftfarbe.de/wp-includes/js/wp-emoji-release.min.js?ver=7.1.2"}}
 </script>
 <script type="module">
@@ -500,3 +500,3 @@
 </body>
 </html>
-<!-- WP Fastest Cache file was created in 1.117 seconds, on 19. September 2026 @ 11:07 -->
+<!-- WP Fastest Cache file was created in 1.623 seconds, on 22. September 2026 @ 19:09 -->
