--- vorher
+++ nachher
@@ -88,4 +88,16 @@
 .modula-best-grid-gallery .fancybox-image {padding-bottom: 2rem !important}
 .modula-best-grid-gallery .is-idle.is-compact .fancybox__footer {animation: none !important}
+.modula-gallery .modula-item,
+.modula-gallery .modula-item-content,
+.modula-gallery .modula-item-overlay,
+.modula-gallery .figc {
+background: transparent !important;
+}
+html body .modula .modula-items .modula-item img.pic { transition: none !important; transform: none !important; }
+@media (min-width: 1025px) {
+.post-inner:has(.modula-gallery) {
+min-height: 930px; /* reale Gesamthöhe der post-inner mit Galerie */
+}
+}
 </style>
 <link href="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/css/front.css?ver=3.0.1" id="modula-css" media="all" rel="stylesheet"/>
@@ -754,3 +766,3 @@
 </body>
 </html>
-<!-- WP Fastest Cache file was created in 1.232 seconds, on 12. September 2026 @ 18:38 -->
+<!-- WP Fastest Cache file was created in 1.268 seconds, on 13. September 2026 @ 10:13 -->
