--- vorher
+++ nachher
@@ -101,5 +101,5 @@
 }
 </style>
-<link href="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/css/front.css?ver=3.0.8" id="modula-css" media="all" rel="stylesheet"/>
+<link href="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/css/front.css?ver=3.0.11" id="modula-css" media="all" rel="stylesheet"/>
 </head>
 <body class="wp-singular page-template-default page page-id-188 wp-theme-hitchcock post single show-preview-titles modula-best-grid-gallery" data-cmplz="1">
@@ -711,9 +711,9 @@
 <script defer="" id="cmplz-cookiebanner-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/complianz-gdpr/cookiebanner/js/complianz.min.js?ver=1788920872">
 </script>
-<script id="modula-isotope-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/isotope.min.js?ver=3.0.8">
-</script>
-<script id="modula-isotope-packery-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/isotope-packery.min.js?ver=3.0.8">
-</script>
-<script id="dompurify-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/purify.min.js?ver=3.0.8">
+<script id="modula-isotope-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/isotope.min.js?ver=3.0.11">
+</script>
+<script id="modula-isotope-packery-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/isotope-packery.min.js?ver=3.0.11">
+</script>
+<script id="dompurify-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/purify.min.js?ver=3.0.11">
 </script>
 <script id="modulaFancybox-js-before">
@@ -743,7 +743,7 @@
 //# sourceURL=modulaFancybox-js-before
 </script>
-<script id="modulaFancybox-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/modula-fancybox.min.js?ver=3.0.8">
-</script>
-<script id="modula-fancybox-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/fancybox.min.js?ver=3.0.8">
+<script id="modulaFancybox-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/modula-fancybox.min.js?ver=3.0.11">
+</script>
+<script id="modula-fancybox-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/fancybox.min.js?ver=3.0.11">
 </script>
 <script id="modula-js-before">
@@ -751,5 +751,5 @@
 //# sourceURL=modula-js-before
 </script>
-<script id="modula-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/jquery-modula.min.js?ver=3.0.8">
+<script id="modula-js" src="https://www.ideetrifftfarbe.de/wp-content/plugins/modula-best-grid-gallery/assets/js/front/jquery-modula.min.js?ver=3.0.11">
 </script>
 <script id="wp-emoji-settings" type="application/json">
@@ -763,3 +763,3 @@
 </body>
 </html>
-<!-- WP Fastest Cache file was created in 3.119 seconds, on 22. September 2026 @ 22:34 -->
+<!-- WP Fastest Cache file was created in 2.907 seconds, on 30. September 2026 @ 5:40 -->
