--- vorher
+++ nachher
@@ -170,9 +170,9 @@
 </script>
 <title>3close Agency - Webdesign by Anil Sardemann</title>
-<!-- All in One SEO 5.0.1.1 - aioseo.com -->
+<!-- All in One SEO 5.0.2 - aioseo.com -->
 <meta content="3close loves to design your web project. Offering customized solutions &amp; direct support for your company's web presence. Come 3close &amp; get in contact." name="description"/>
 <meta content="max-image-preview:large" name="robots"/>
 <link href="https://www.3close.de/" rel="canonical"/>
-<meta content="All in One SEO (AIOSEO) 5.0.1.1" name="generator"/>
+<meta content="All in One SEO (AIOSEO) 5.0.2" name="generator"/>
 <meta content="en_US" property="og:locale"/>
 <meta content="3close Agency • webdesign by anil sardemann" property="og:site_name"/>
