--- vorher
+++ nachher
@@ -66,5 +66,5 @@
 <link as="script" crossorigin="" fetchpriority="low" href="https://www.3close.de/wp-content/plugins/preloader-plus//assets/js/preloader-plus.min.js" rel="preload" type="text/javascript"/>
 <link as="style" crossorigin="" fetchpriority="low" href="https://www.3close.de/wp-content/plugins/preloader-plus/assets/css/preloader-plus.min.css" rel="preload" type="text/css"/>
-<!-- This site is optimized with the Yoast SEO plugin v28.4 - https://yoast.com/product/yoast-seo-wordpress/ -->
+<!-- This site is optimized with the Yoast SEO plugin v28.5 - https://yoast.com/product/yoast-seo-wordpress/ -->
 <meta content="Find additional information for when to use a Dark Mode and why. Let 3close consult you in finding the best solution for your website." name="description"/>
 <link href="https://www.3close.de/answers/2024/01/24/light-and-dark-vs-black-and-white/" rel="canonical"/>
