--- vorher
+++ nachher
@@ -70,5 +70,5 @@
 <script id="hitchcock_global-js" src="https://www.ideetrifftfarbe.de/wp-content/themes/hitchcock/assets/js/global.js?ver=2.0.2">
 </script>
-<meta content="WordPress 7.1" name="generator"/>
+<meta content="WordPress 7.1.1" name="generator"/>
 <link href="https://www.ideetrifftfarbe.de/datenschutzerklaerung/" rel="canonical"/>
 <link href="https://www.ideetrifftfarbe.de/?p=3" rel="shortlink"/>
@@ -489,5 +489,5 @@
 </script>
 <script id="wp-emoji-settings" type="application/json">
-{"baseUrl":"https://s.w.org/images/core/emoji/17.0.2/72x72/","ext":".png","svgUrl":"https://s.w.org/images/core/emoji/17.0.2/svg/","svgExt":".svg","source":{"concatemoji":"https://www.ideetrifftfarbe.de/wp-includes/js/wp-emoji-release.min.js?ver=7.1"}}
+{"baseUrl":"https://s.w.org/images/core/emoji/17.0.2/72x72/","ext":".png","svgUrl":"https://s.w.org/images/core/emoji/17.0.2/svg/","svgExt":".svg","source":{"concatemoji":"https://www.ideetrifftfarbe.de/wp-includes/js/wp-emoji-release.min.js?ver=7.1.1"}}
 </script>
 <script type="module">
@@ -498,3 +498,3 @@
 </body>
 </html>
-<!-- WP Fastest Cache file was created in 1.177 seconds, on 16. September 2026 @ 19:10 -->
+<!-- WP Fastest Cache file was created in 1.289 seconds, on 18. September 2026 @ 1:00 -->
