--- vorher
+++ nachher
@@ -92,4 +92,16 @@
 .modula-best-grid-gallery .fancybox-image {padding-bottom: 2rem !important}
 .modula-best-grid-gallery .is-idle.is-compact .fancybox__footer {animation: none !important}
+.modula-gallery .modula-item,
+.modula-gallery .modula-item-content,
+.modula-gallery .modula-item-overlay,
+.modula-gallery .figc {
+background: transparent !important;
+}
+html body .modula .modula-items .modula-item img.pic { transition: none !important; transform: none !important; }
+@media (min-width: 1025px) {
+.post-inner:has(.modula-gallery) {
+min-height: 930px; /* reale Gesamthöhe der post-inner mit Galerie */
+}
+}
 </style>
 </head>
@@ -486,3 +498,3 @@
 </body>
 </html>
-<!-- WP Fastest Cache file was created in 0.851 seconds, on 12. September 2026 @ 18:57 -->
+<!-- WP Fastest Cache file was created in 1.176 seconds, on 13. September 2026 @ 10:36 -->
